src/ui/preview/Playground.tsx

```diff
@@ -55,9 +56,26 @@ async function initPlayground(
 	slug: string,
 	blogName: string
 ): Promise< PlaygroundClient > {
+	const mountDescriptor: MountDescriptor = {
+		device: {
+			type: 'opfs',
+			path: '/try-wp-' + slug,
```

Comment:
@psrpinto Should we nest this one level? `/try-wp-sites/{slug}`

Reply:
I like that. Hopefully Playground is able to deal with subdirs.